docs/setup-guide/cluster-up-kube-up.md

```diff
@@ -0,0 +1,58 @@
+# Setting up Arktos scale out environment in Google Cloud
+
+This document gives brief introduction on how to set up Arktos scale out environment in Google Cloud.
+
+Step 1. Request a VM in Google Cloud Platform, recommended configuration: ubuntu 18.04+, 8 cpu or above, disk size 200GB or up.
+
+Step 2. Run "gcloud version" to encure the Google Gloud DSK is updated (recommended Google Cloud SDK version is 298.0.0 and up). Please refer to https://cloud.google.com/sdk/docs/downloads-apt-get or https://cloud.google.com/sdk/docs/downloads-versioned-archives to upgrade your google cloud SDK. 
+
+Step 3. Follow [dev environment setup instruction](setup-dev.md) to set up developer environment on this newly created GCE instance.
+
+Step 4. Build prepare:
+
+```bash
+cd ~/go/src/arktos
+make clean
+make quick-release
+```
+
+Step 5. Set Arktos cluster configuration. The following are the typical environment variables and values used in Arktos 2TP 2RP scale out cluster with Mizar CNI Plugin:
+```bash
+export NUM_NODES=2                # number of nodes for each RP
+export SCALEOUT_TP_COUNT=2        # number of tenant partitions to set up
+export SCALEOUT_RP_COUNT=2        # number of resource partitions to set up
+export RUN_PREFIX=<fill-in-your-cluster-name>       # cluster name
+
+export MASTER_DISK_SIZE=200GB       # disk size for master machine to run API server, scheduler, controller manager, etc.
+export MASTER_ROOT_DISK_SIZE=200GB 
+export KUBE_GCE_ZONE=us-central1-b  # the location of the new arktos cluster
+export MASTER_SIZE=n1-highmem-32    # machine size for masters
+export NODE_SIZE=n1-highmem-16      # machine size for workers
+export NODE_DISK_SIZE=200GB         # disk size for workers
+export GOPATH=$HOME/go 
+export KUBE_GCE_ENABLE_IP_ALIASES=true 
+export KUBE_GCE_PRIVATE_CLUSTER=true 
+export CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=${RUN_PREFIX} KUBE_GCE_NETWORK=${RUN_PREFIX} ENABLE_KCM_LEADER_ELECT=false ENABLE_SCHEDULER_LEADER_ELECT=false SHARE_PARTITIONSERVER=false LOGROTATE_FILES_MAX_COUNT=50 LOGROTATE_MAX_SIZE=200M KUBE_ENABLE_APISERVER_INSECURE_PORT=true KUBE_ENABLE_PROMETHEUS_DEBUG=true KUBE_ENABLE_PPROF_DEBUG=true
+export SCALEOUT_CLUSTER=true 
+
+export NETWORK_PROVIDER=mizar
```

Comment:
Suggest to merge these export to 2 or 3. Here is the one example that team are using for daily run:
```
export NUM_NODES=2 SCALEOUT_CLUSTER=true SCALEOUT_TP_COUNT=2 SCALEOUT_RP_COUNT=2 RUN_PREFIX=<fill-in-your-cluster-name>
export MASTER_DISK_SIZE=500GB MASTER_ROOT_DISK_SIZE=500GB KUBE_GCE_ZONE=us-west2-b MASTER_SIZE=n1-highmem-32 NODE_SIZE=n1-highmem-16 NODE_DISK_SIZE=500GB GOPATH=$HOME/go KUBE_GCE_ENABLE_IP_ALIASES=true KUBE_GCE_PRIVATE_CLUSTER=true CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=${RUN_PREFIX} KUBE_GCE_NETWORK=${RUN_PREFIX} ENABLE_KCM_LEADER_ELECT=false ENABLE_SCHEDULER_LEADER_ELECT=false ETCD_QUOTA_BACKEND_BYTES=8589934592 SHARE_PARTITIONSERVER=false LOGROTATE_FILES_MAX_COUNT=200 LOGROTATE_MAX_SIZE=200M KUBE_ENABLE_APISERVER_INSECURE_PORT=true KUBE_ENABLE_PROMETHEUS_DEBUG=true KUBE_ENABLE_PPROF_DEBUG=true TEST_CLUSTER_LOG_LEVEL=--v=2 HOLLOW_KUBELET_TEST_LOG_LEVEL=--v=2 GCE_REGION=us-west2-b NETWORK_PROVIDER=mizar
```

Reply:
> Suggest to merge these export to 2 or 3. Here is the one example that team are using for daily run:
> 
> ```
> export NUM_NODES=2 SCALEOUT_CLUSTER=true SCALEOUT_TP_COUNT=2 SCALEOUT_RP_COUNT=2 RUN_PREFIX=<fill-in-your-cluster-name>
> export MASTER_DISK_SIZE=500GB MASTER_ROOT_DISK_SIZE=500GB KUBE_GCE_ZONE=us-west2-b MASTER_SIZE=n1-highmem-32 NODE_SIZE=n1-highmem-16 NODE_DISK_SIZE=500GB GOPATH=$HOME/go KUBE_GCE_ENABLE_IP_ALIASES=true KUBE_GCE_PRIVATE_CLUSTER=true CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=${RUN_PREFIX} KUBE_GCE_NETWORK=${RUN_PREFIX} ENABLE_KCM_LEADER_ELECT=false ENABLE_SCHEDULER_LEADER_ELECT=false ETCD_QUOTA_BACKEND_BYTES=8589934592 SHARE_PARTITIONSERVER=false LOGROTATE_FILES_MAX_COUNT=200 LOGROTATE_MAX_SIZE=200M KUBE_ENABLE_APISERVER_INSECURE_PORT=true KUBE_ENABLE_PROMETHEUS_DEBUG=true KUBE_ENABLE_PPROF_DEBUG=true TEST_CLUSTER_LOG_LEVEL=--v=2 HOLLOW_KUBELET_TEST_LOG_LEVEL=--v=2 GCE_REGION=us-west2-b NETWORK_PROVIDER=mizar
> ```

The reason that I split them is for commenting purpose. This is showing as example for other devs to use in their own setting. Prefer not to use the consolidated command.